packages/components/src/molecules/ProductThumbnail/ProductThumbnail.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+
+export interface ProductThumbnailProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+}
+
+const ProductThumbnail = forwardRef<HTMLDivElement, ProductThumbnailProps>(
```

Comment:
@artursantiago I talked to @renatamottam and we believe this component isn't needed in the components package since it's not being reused elsewhere.
We're considering moving it to `core/src/components/reviews/ProductThumbnail`.  Does that make sense to you?